src/apollo/mutations.graph.ts

```diff
@@ -1,9 +1,21 @@
 import { gql } from '@apollo/client';
 
+// Fragment for all of these
+// Have this mutation with same data requirement as completed task
 export const CREATE_TASK = gql`
     mutation createTask($input: CreateTaskInput!) {
         createTask(input: $input) {
             id
+            name
```

Comment:
Mutations were modified to get the values so they can be stored/updated/deleted in the cache